projects/packages/forms/src/dashboard/components/empty-trash-button/index.tsx

```diff
@@ -0,0 +1,128 @@
+/**
+ * External dependencies
+ */
+import jetpackAnalytics from '@automattic/jetpack-analytics';
+import apiFetch from '@wordpress/api-fetch';
+import { Button, __experimentalConfirmDialog as ConfirmDialog } from '@wordpress/components'; // eslint-disable-line @wordpress/no-unsafe-wp-apis
+import { useEntityRecords, store as coreStore } from '@wordpress/core-data';
+import { dispatch } from '@wordpress/data';
+import { useState, useCallback } from '@wordpress/element';
+import { __, _n, sprintf } from '@wordpress/i18n';
+import { trash } from '@wordpress/icons';
+import { store as noticesStore } from '@wordpress/notices';
+/**
+ * Internal dependencies
+ */
+import { store as dashboardStore } from '../../store';
+
+type CoreStore = typeof coreStore & {
+	invalidateResolutionForStore: ( store: typeof dashboardStore ) => void;
+};
+
+/**
+ * Renders a button to empty form responses.
+ *
+ * @return {JSX.Element} The empty trash button.
+ */
+const EmptyTrashButton = (): JSX.Element => {
+	const [ isOpen, setOpen ] = useState( false );
+	const [ isLoading, setIsLoading ] = useState( false );
+
+	const { createSuccessNotice, createErrorNotice } = dispatch( noticesStore );
+	const { invalidateResolutionForStore } = dispatch( coreStore ) as unknown as CoreStore;
+
+	const { totalItems } = useEntityRecords( 'postType', 'feedback', {
+		status: 'trash',
+	} );
+
+	const isEmpty = totalItems === 0;
```

Comment:
This causes a little bug: when loading the `jetpack-forms-admin` page, the "Empty trash" button is always enabled for a moment while the `/feedback` data are loading, and gets disabled only when the response arrives. That's because `totalItems === undefined` during the load. The `isEmpty` condition should also cover this case and enable the button only when `totalItems` is really `> 0`.